examples/unresolved-cfd-dem/gas-solid-spouted-cylinder-bed/gas-solid-spouted-bed.prm

```diff
@@ -0,0 +1,298 @@
+# Listing of Parameters
+#----------------------
+
+set dimension = 3
+
+#---------------------------------------------------
+# Simulation Control
+#---------------------------------------------------
+
+subsection simulation control
+  set method               = bdf1
+  set number mesh adapt    = 0
+  set output name          = result
+  set output frequency     = 10
+  set startup time scaling = 0.6
+  set time end             = 5
+  set time step            = 0.001
+  set subdivision          = 1
+  set log precision        = 10
+  set output path          = ./output/
+end
+
+#---------------------------------------------------
+# Restart
+#---------------------------------------------------
+
+subsection restart
+  set checkpoint = true
+  set frequency  = 100
+  set restart    = false
+  set filename   = case
+end
+
+#---------------------------------------------------
+# FEM
+#---------------------------------------------------
+
+subsection FEM
+  set velocity order = 1
+  set pressure order = 1
+end
+
+#---------------------------------------------------
+# Physical Properties
+#---------------------------------------------------
+
+subsection physical properties
+  subsection fluid 0
+    set kinematic viscosity = 0.0000181
+    set density             = 1.0
+  end
+end
+
+#---------------------------------------------------
+# Mesh
+#---------------------------------------------------
+
+subsection mesh
+  set type                                = gmsh
+  set file name                           = ./mesh/cylinder-spouted-bed.msh
+  set expand particle-wall contact search = false
+end
+
+#---------------------------------------------------
+# Void Fraction
+#---------------------------------------------------
+
+subsection void fraction
+  set mode                         = qcm
+  set qcm sphere equal cell volume = true
+  set read dem                     = true
+  set dem file name                = dem
+  set bound void fraction          = false
+end
+
+#---------------------------------------------------
+# CFD-DEM
+#---------------------------------------------------
+
+subsection cfd-dem
+  set grad div                      = true
+  set void fraction time derivative = true
+  set drag force                    = true
+  set buoyancy force                = true
+  set shear force                   = true
+  set pressure force                = true
+  set saffman lift force            = false
+  set drag model                    = rong
+  set coupling frequency            = 100
+  set implicit stabilization        = false
+  set grad-div length scale         = 0.005
```

Comment:
If you are changing this variable here, I would explain in the example rst file why you are setting the variable this way. Why did you chose this value.